app_configs.py

```diff
@@ -19,7 +19,7 @@
 # Sets Dash debug which hides and shows Dash debug menu.
 # Set to True if developing and False if demoing.
 # App should be restarted to see change.
-DEBUG = True
+DEBUG = False
```

Comment:
Easy to forget to change back! 😄 

```suggestion
DEBUG = True
```

Reply:
Don't we want it to be False for the demo-ing?

Reply:
Oh, sorry, that's true. 🤦‍♂️